app/forms/candidate_interface/restructured_work_history/job_form.rb

```diff
@@ -81,25 +82,10 @@ def save(application_form)
           end_date_unknown: end_date_unknown,
           currently_working: currently_working,
           relevant_skills: relevant_skills,
+          details: set_details_field,
         )
       end
 
-      def blank_form?
-        [
-          role,
-          organisation,
-          commitment,
-          start_date_day,
-          start_date_month,
-          start_date_year,
-          end_date_day,
-          end_date_month,
-          end_date_year,
-          currently_working,
-          relevant_skills,
-        ].all?(&:blank?)
-      end
-
```

Comment:
Unrelated to the commit message, suggest putting this deletion into its own commit.

Reply:
Yea good shout. Was being lazy!